articles/clients/index.md

```diff
@@ -4,186 +4,167 @@ description: Explains the basics of creating and using Auth0 Clients.
 
-![](/media/articles/applications/applications-single-app-active-users.png)
+![Example diagram](/media/articles/applications/applications-traditional.png)
 
-In the following month, John and Mary start using the company's collaboration application. This means that they'll become active users for this second application. All together, this means that there are a total of 5 active social users for the month of March:
+For a regular web application, all you need to do is create a new Client in Auth0.
 
- * Todo List: 3 active users
- * Collaboration Application: 2 active users
+### Mobile Clients, a Single Page Application, and an REST API
 
-![](/media/articles/applications/applications-multi-app-active-users.png)
+![Example diagram](/media/articles/applications/multiple-clients.png)
 
-## Sample Clients
+This example is a timesheet application that utilizes:
 
-The following are high-level overviews of sample Clients using a variety of technologies.
+* A REST API that receives timesheet entries from users.
+* A Single Page Application (SPA) used to log timesheets entries and send them to the REST API.
+* Mobile apps capable of running on several types of devices, used to log timesheets entries and send them to the REST API.
 
-> One client is comprised of an ID-secret pair. If multiple clients (for example, one runs on iOS and one runs on Android) share the same Client ID-secret pair, Auth0 considers them to be a single Client.
+In order to configure this in Auth0 you would create:
+- A client for the SPA.
+- A client for the iOS mobile app.
+- A client for the Android mobile app.
+- A client for the Windows mobile app.
+- An API.
 
-### Regular Web Application
+For more details on APIs and how to configure one refer to: [APIs](/apis).
 
-![](/media/articles/applications/applications-traditional.png)
+### Multiple Services and APIs
 
-For a regular web application, all you need to do is create a new Client in Auth0.
+![Example diagram](/media/articles/applications/multiple-apis.png)
 
-### Mobile Clients, a Single Page Application, and an REST API
+This example is for a Portal (HTML5 SPA) that communicates with four APIs.
 
-![](/media/articles/applications/applications-multiple-single-logical.png)
+In order to configure this in Auth0 you would create:
+- A client for the SPA.
+- An API for the Portal REST API.
+- An API for the Documents API.
+- An API for the Invoices API.
+- An API for the Contacts API.
 
-This example is a timesheet application that utilizes:
+For more details on APIs and how to configure one refer to: [APIs](/apis).
 
-* A REST API;
-* A Single Page Application (SPA) hosted on a server different from the one hosting the REST API;
-* Mobile apps capable of running on several types of devices.
 
-From a technical standpoint, the above comprises at least three Clients, due to their differing language, deployment model, and so on. However, for Auth0, this is **one** Client application that shares an ID-secret pair.
+## Auditing
```

Comment:
should we leave this here? maybe we can point to a much detailed doc where we discuss logs? wdyt?